packages/powersync_attachments_stream/lib/src/abstractions/local_storage.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:typed_data';
+
+abstract class AbstractLocalStorageAdapter {
```

Comment:
Let's make this an interface class and just name it `LocalStorageAdapter`:

```suggestion
abstract interface class LocalStorageAdapter {
```